pages/about.html

```diff
@@ -0,0 +1,126 @@
+    <link rel="stylesheet" media="screen and (min-width: 768px)" href="../css/wide-screen.css">
+    <link rel="stylesheet" media="screen and (min-width: 768px)" href="../css/about-wide-screen.css">
+    <title>About</title>
+</head>
+
+<body>
+    <section class="header-background">
+        <header class="header">
+            <div class="social-media">
+                <a href="#" class="social-media-link">
+                    <img src="" alt="">
+                </a>
+                <a href="" class="social-media-link">
+                    <img src="" alt="">
+                </a>
+                <a href="" class="last" class="social-media-link"> English </a>
+            </div>
+            <nav class="nav-bar">
+                <a href="../index.html" class="logo"> <img src="../img/logo_muda.jpg" alt="" width="80px" height="70px"></a>
+                <ul class="nav-menu">
+                    <li class="nav-item nav-item-1">
+                        <a href="about.html" class="nav-link"> About</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#" class="nav-link"> Program</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#" class="nav-link"> Join</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#" class="nav-link"> Sponsor</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#" class="nav-link"> News</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#" class="nav-link"> CC Campaign</a>
+                    </li>
+                    <div class="close-btn">
+                        <img src="../img/close.png" alt="Close burger menu">
+                    </div>
+                </ul>
+                <div class="burger-menu">
+                    <a href="#" class="burger-menu-btn">
+                        <img src="../img/menu.png" alt="Open menu" width="24px" height="17.5px">
+                    </a>
+                </div>
+            </nav>
+        </header>
+        <div class="header-text">
+            <h1 class="title"> Projecto Muda Solidary Christmas 2022 </h1>
+            <P class="primary-text"> Projecto Muda is a movement that started back in 2018 and it aims to raise young people awareness on social problems. Project Muda believes that anyone can help and you do not need too much to do so. This movement has been able to help many families and the 2022 Edition won't be any different. </P>
+            <div class="meeting-date-place">
+                <h2 class="date"> Please contact us per Email for any further questions about CC Global Summit 2022 </h2>
+                <a href="#" class="place"> projectomuda@gmail.com</a>
+            </div>
+        </div>
+    </section>
```

Comment:
- [ ] For making your project use semantic tags correctly and the HTML markup is well structured and follows a clear hierarchy, I recommend you to use header with header tag and headline section with section tag . You used for both header and headline section with the same section tag, Please fix it.